testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/settings/SettingsMenuActivity.kt

```diff
@@ -109,7 +114,10 @@ class SettingsMenuActivity : BaseActivity() {
         binding.editParametersUrl.setText(settings.urlParameters)
         binding.switchPreviewMode.isChecked = settings.isPreviewMode
         binding.switchSignatureVerification.isChecked = settings.requireSignatureVerification
-
+        binding.switchProdVersion.isChecked = settings.isProdVersionEnabled
+        if(BuildConfig.BUILD_TYPE == BuildVariant.RELEASE.name){
```

Comment:
This toggle switch needs to be available in both the Release and QA builds.

Reply:
@corycaywood we have `debug`, `release`, `rc` and `staging`.. `release` uses the prod url and keys, so i thought it be better to hide that in release build, so if i show it in release build as well then what will happen if they toggle off/on the switch?